index.bs

```diff
@@ -1568,6 +1580,54 @@ for accelerometer sensor is given below.
     };
 </pre>
 
+<h3 id="feature-policy-api">Extending the Feature Policy API</h3>
+
+An implementation of the {{Sensor}} interface for each [=sensor type=] has a
+[=policy-controlled feature=] that controls whether or not this implementation
+can be used in a document.
+
+The [=policy-controlled feature|feature=]'s [=default allowlist=] is
+<code>["self"]</code>.
+
+Note: The [=default allowlist=] of <code>["self"]</code> allows {{Sensor}} interface
+implementation usage in same-origin nested frames but prevents third-party content
+from [=sensor readings=] access.
+
+The [=feature name=] for this [=policy-controlled feature|feature=]
+is [=sensor feature name=].
+
+A [=Low-level=] {{Sensor|sensor}} may use its interface name as a [=sensor feature name=],
+for instance, "gyroscope" or "accelerometer". Unless the [=extension specification=] tells
+otherwise, the [=sensor feature name=] matches the [=sensor type|type=]-associated
+{{PermissionName}}.
+
+<div class="example html">
+Third-party Accelerometer usage is selectively enabled by adding [=allow attribute=]
```

Comment:
The accelerometer feature is selectively enabled for third-party origin by adding ...